llm/dbrx/README.md

```diff
@@ -0,0 +1,177 @@
+# Databricks DBRX: A State-of-the-Art Open LLM
+
+<p align="center">
+  <img src="https://www.databricks.com/en-blog-assets/static/2fe1a0af1ee0f6605024a810b604079c/dbrx-blog-header-optimized.png" alt="DBRX Blog Header" height="200">
+</p>
+
+[DBRX](https://www.databricks.com/blog/introducing-dbrx-new-state-art-open-llm) is an open, general-purpose LLM created by Databricks. It uses a mixture-of-experts (MoE) architecture with 132B total parameters of which 36B parameters are active on any input.
+
+In this recipe, you will serve `databricks/dbrx-instruct` on your own infra  -- existing Kubernetes cluster or cloud VMs -- with one command.
+
+## Prerequisites
+
+- Go to the [HuggingFace model page](https://huggingface.co/databricks/dbrx-instruct) and request access to the model `databricks/dbrx-instruct`.
+- Check that you have installed SkyPilot ([docs](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html)).
+- Check that `sky check` shows clouds or Kubernetes are enabled.
+
+## SkyPilot YAML
+
+See the full YAML file [here](https://github.com/skypilot-org/skypilot/tree/master/llm/dbrx/dbrx.yaml).
+
```

Comment:
Could we add the YAML inline and collapsed here?

Reply:
Yep, done.